Classes/Controller/FeUsersController.php

```diff
@@ -117,7 +116,9 @@ public function renderAjax(ServerRequestInterface $request): Response
 			if (isset($row['usergroup'])) {
 				$usergroups = [];
 				foreach (explode(',', $row['usergroup']) as $usergroupUid) {
```

Comment:
I would also update this to be

```
use \TYPO3\CMS\Core\Utility\GeneralUtility;

GeneralUtility::trimExplode()
```

instead of using the `explode`

Reply:
I know this is strictly not you, but while you're here :)